README.md

```diff
@@ -24,7 +24,8 @@ It supports English, French, and most **Latin languages**, but not Cyrillic or C
 
 TOC (generated by this tool, duh, using the `github` profile):
 
-<!-- TOC start -->
+<!-- TOC start (generated with https://github.com/derlin/bitdowntoc) -->
```

Comment:
I would prefer the *.io URL, for us to click on the link and regenerate directly.

But I confess that github.com click and, there, github.io click, is not that difficult.

Reply:
I hesitated as well,  but there is also a CLI (jar) available. Moreover, I am thinking of maybe moving the github.io website to one of my domains (if you have an opinion, I am all ears). The GitHub URL is thus less likely to change. The implementation makes it possible to change our minds later though :)

Reply:
OK that makes sense, let's leave it that way ✌️